test/simple/test-tls-psk-client.js

```diff
@@ -0,0 +1,166 @@
+// Copyright & License details are available under JXCORE_LICENSE file
```

Comment:
in this test, openssl s_server is used as a psk host; the `test/simple/test-tls-psk-circuit` provides a mixed set of ciphers.